index.html

```diff
@@ -1,61 +1,305 @@
+        </div>
+
+        <div class="middle-section-nav-bars">
+          <ul class="technology">
+            <li>Html</li>
+            <li>css</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+
+        <button class="project-link" type="submit">See Project</button>
+      </div>
+    </section>
+
+    <section class="about-myself">
+      <h1 class="header">About Myself</h1>
+
+      <div class="header-text">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      </div>
+
+      <button class="about-myself-link" type="submit">LET'S CONNECT</button>
+
+      <div class="about-myself-icons">
+        <img src="photo/logos/github.jpg" alt="" />
+        <img src="photo/navs/Linkedin-icon.jpg" alt="" />
+        <img src="photo/logos/angellist.jpg" alt="" />
+        <img src="photo/logos/twitter.jpg" alt="" />
+        <img src="photo/logos/medium.jpg" alt="" />
+      </div>
+
+      <button class="resume" type="submit">Get my resume</button>
+
+      <div class="language">
+        <button class="language" type="submit">Languages</button>
+
+        <div>
+          <img src="photo/ben/Dropdown.png" alt="" />
+        </div>
+      </div>
+
+      <ul class="about-myself-ul">
+        <li class="java">
+          <img src="photo/logos/js-image.png" alt="javascript" />
+          <span>JavaScript</span>
+        </li>
+
+        <li class="html">
+          <img src="photo/logos/html-image.png" alt="html" />
+          <span>HTML</span>
+        </li>
+
+        <li class="css">
+          <img src="photo/logos/css-image.png" alt="css" />
+          <span>CSS</span>
+        </li>
+      </ul>
+
+      <ul class="frameworks">
+        <li>
+          <h3>Frameworks</h3>
+          <img src="photo/ben/capture-side.PNG" alt="" />
+        </li>
+      </ul>
+
+      <ul class="skills">
+        <li>
+          <h3>Skills</h3>
+          <img src="photo/ben/capture-side.PNG" alt="" />
+        </li>
+      </ul>
+    </section>
+
+    <section class="form-container">
+      <div class="form-text">
+        <h1>Contact me</h1>
+      </div>
+      <div class="paragragh">
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+
+      <div class="contact-form">
+        <form
+          action="https://formspree.io/f/xbjwvedl"
+          method="post"
+          name="form-collection"
+          class="my-form-collection"
+        >
+          <ul>
+            <li><input type="text" placeholder="Duncan" maxlength="30" required/></li>
+            <li><input type="email" placeholder="njaugitahi@gmail.com" required /></li>
```

Comment:
- [ ] It will be better if you add a good placeholder instead of your name and email to not confuse the recruiter when trying to connect with you.
```
Good example:
name: your name
email: your email
```